### ISSUE TITLE
Support LAB colors

### ISSUE THREAD

**Issue by azat-io** [GoogleChrome/lighthouse#15125]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://eslint-plugin-perfectionist.azat.io

### What happened?

I use LAB colors on my website. `lab()`, `lch()`, `oklab()`, and `oklch()`

<img width="610" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/5698350/d9e6871d-857a-44a6-a8af-3971fc377c37">

However, when using colors, Lighthouse cannot correctly calculate the contrast of the colors. The check "Background and foreground colors have a sufficient contrast ratio" cannot be performed.

### What did you expect?

Availability rate is calculated correctly.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

10.0.1

### Chrome Version

113.0.5672.126

### OS

Mac

**Comment by adamraine** [comment]
This issue has been fixed in the latest version of Lighthouse.

**Comment by brycewray** [comment]
...so apparently Chrome 114.0.5735.106 isn't running that version, because I get the same report if I use even a small number of OKLCH colors. If I replace them with HSL, the accessibility score goes from "?" to "100."

**Comment by adamraine** [comment]
I was mistaken, the change has landed but we haven't released it yet. Planning to do that today.
